src/timePicker/index.tsx

```diff
@@ -41,12 +41,15 @@ export const getTodayDate = () => {
 
 /**
  * Format given dateTime with `Date#toLocaleTimeString`.
+ * Node 10.x.x uses a new implementation for date formatting,
+ * so we don't rely on it and check the locale for 24-hour format.
```

Comment:
Please add a comment to TimePickerProps#renderTime:
'Custom renderTime. If not provided a default one without proper i18n will be used.'